### ISSUE TITLE
Grafana: changed query field

### ISSUE THREAD

**Issue by tdorsey** [galexrt/container-node_exporter-textfiles#1]
The temp fields reported by smartctl do not match the grafana query. 

Smartctl reports `smartmon_temperature_celsius_raw_value`
The graph is looking for `smartmon_airflow_temperature_cel_value`

smartctl version 6.4 2014-10-07 r4002 [x86_64-linux-4.19.0-0.bpo.2-amd64]
image digest: 7c54c3be8aa5

Happy to make the PR myself, just not sure if it's a global issue or not

**Comment by galexrt** [comment]
@tdorsey Thanks for raising the issue!

I have gone ahead and updated the Dockerfile to use `debian:buster-slim` and updated the `smartmon.sh` script.

Could you please verify that continue to get the `smartmon_temperature_celsius_raw_value` metrics with the latest image `v1.1.0` (quay image is already built, Docker Hub still needs a few more minutes).

A potential fix could be to use the `and` operator (in my case it seems to work as expected):
```
avg(smartmon_temperature_celsius_raw_value{instance=~"$instance"} and smartmon_airflow_temperature_cel_value) by (instance, disk)
```
(https://prometheus.io/docs/prometheus/latest/querying/operators/#logical-set-binary-operators)
Would you be up to open a PR changing the affected queries to use `and` in them as shown in the example?

**Comment by tdorsey** [comment]
Yeah, that's no problem, although I can't guarantee an ETA.

Keeping in mind that I know next to nothing about prometheus vectors, do you know why an `or` instead of `and` fails to return data? I'd assume that if smartmon ever returned both airflow* and raw* that the graph would have duplicated data, but using an or operator fails to return either set...

**Comment by galexrt** [comment]
I have looked into the whole thing a bit further, seems that different HDDs might report a different set of SMART values.
In my K8S cluster with the v1.1.0 image (and before that) I'm getting both metrics (`smartmon_temperature_celsius_raw_*` and `smartmon_airflow_temperature_*`).
![2019-09-29-201954_1273x947_scrot](https://user-images.githubusercontent.com/3718398/65837577-53ab9a80-e2f9-11e9-8447-e85059a9a01d.png)

What do you think about adding keeping the `Disk Temperature` and adding a new panel next to it with the airflow temps?
So like this [Smartmon Textfile-1569782184745.txt](https://github.com/galexrt/docker-node_exporter-smartmon/files/3667920/Smartmon.Textfile-1569782184745.txt)
(I had to rename the file to end with `.txt` because GitHub doesn't allow `.json` uploads)
This isn't a perfect solution, as one graph might be empty depending on the SMART values exposed by the disks, though at least from my perspective it would be a good compromise for the dashboard.

**Comment by tdorsey** [comment]
That works for me. I’ll work that into the PR. 

This does mean that the chart json would move into this repo though. I don’t know of a way to collaborate on a chart that’s been uploaded to grafana. Is that ok?

**Comment by galexrt** [comment]
Yeah, there isn't really a good way. Moving the chart in the repo is fine.

**Comment by galexrt** [comment]
@tdorsey ping, any updates on this issue?

**Comment by tdorsey** [comment]
@galexrt Sorry for the delay on this. The version of grafana I'm working with has changed json schema formats significantly in the interim, but this version is working for me. Let me know if you see issues

**Comment by galexrt** [comment]
@tdorsey As written in the PR, just update the existing dashboard json file. Please add the Grafana version requirement to the README Grafana section.
